tests/common/debug.py

```diff
@@ -85,3 +88,31 @@ def condition(x):
         settings=settings,
         random=random,
     )
+
+
+def assert_no_examples(strategy, condition=None):
+    @Settings(perform_health_check=False, max_examples=1, max_iterations=100)
+    @given(strategy)
+    def test(s):
+        if condition is not None:
+            assume(condition(s))
+
+    with pytest.raises(Unsatisfiable):
```

Comment:
try/except Unsatisfiable/else assert False would be enough to remove the pytest dependency here.

Reply:
Is that a thing we need/want to do?

Reply:
Well, trying to import pytest made the check-nose and check-unicode jobs fail [in the last build](https://travis-ci.org/HypothesisWorks/hypothesis-python/builds/274600350), so I would!

Reply:
Ah, yes, right.